crowbar_framework/app/controllers/installers_controller.rb

```diff
@@ -42,27 +46,42 @@ def status
   # Provides the restful api call for
   # /installer/start 	POST 	triggers installation
   def start
-    if Crowbar::Installer.successful?
-      respond_to do |format|
-        format.json { render json: Crowbar::Installer.status }
-        format.html { redirect_to installer_url }
+    header = :ok
+    msg, msg_type = "", :alert
+
+    if params[:force]
+      ret = Crowbar::Installer.install!
+      case ret[:status]
+      when 501
+        header = :not_implemented
+        msg = ret[:msg]
       end
     else
-      # the shell Process will be spawned in the background and therefore has
-      # not a direct return value which we can use here
-      if Crowbar::Installer.installing?
-        flash[:notice] = I18n.t(".installers.start.installation_ongoing")
+      if Crowbar::Installer.successful?
+        header = :gone
       else
-        ret = Crowbar::Installer.install
-        case ret[:status]
-        when 501
-          flash[:alert] = ret[:msg]
+        if Crowbar::Installer.installing?
+          header = :im_used
+          msg = I18n.t(".installers.start.installation_ongoing")
+          msg_type = :notice
+        else
+          ret = Crowbar::Installer.install
+          case ret[:status]
```

Comment:
Metrics/BlockNesting: Avoid more than 3 levels of block nesting. (https://github.com/bbatsov/ruby-style-guide#three-is-the-number-thou-shalt-count)